tools/lox_gen/src/rotational_elements.rs

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+use crate::Data;
+use proc_macro2::{Ident, TokenStream};
+use quote::{format_ident, quote};
+
+/// Holds the rotational elements for a given body.
+///
+/// May be parsed directly from PCK data and knows how to represent its code and test components as
+/// streams of tokens.
+pub struct RotationalElements<'a> {
+    id: i32,
+    ident: &'a Ident,
+    right_ascension: (f64, f64, f64),
+    declination: (f64, f64, f64),
+}
+
+impl<'a> RotationalElements<'a> {
+    pub(crate) fn parse(id: i32, ident: &'a Ident, data: &Data) -> Option<Self> {
+        let right_ascension_key = format!("BODY{}_POLE_RA", id);
+        let right_ascension = get_polynomial_coefficients(&right_ascension_key, data)?;
```

Comment:
This will likely become a Result if we refactor the error handling to avoid panics.